src/controller/favoriteHotel.controller.js

```diff
@@ -0,0 +1,67 @@
+import favUserHotel from "../model/favoriteHotel.model.js";
+import { response } from "express";
+
+export const favoritesGet = async (req, res = response) => {
+  const { limite, desde } = req.query;
+  const query = {};
+
+  const [total, favorites] = await Promise.all([
+    favUserHotel.countDocuments(query),
+    favUserHotel.find(query).skip(Number(desde)).limit(Number(limite)),
+  ]);
```

Comment:
porfavor cambiar uso de query param `desde` a `page` y trabajar en ingles siempre! Ademas, debemos filtrar por elementos con `tp_status`  ACTIVE. 
```suggestion
export const favoritesGet = async (req, res = response) => {
  const { limite, page } = req.query;
  const query = { tp_status: "ACTIVE"};

  const [total, favorites] = await Promise.all([
    favUserHotel.countDocuments(query),
    favUserHotel.find(query).skip(Number(desde)).limit(Number(limite)),
  ]);
```